packages/backend/src/apps/postman/common/throw-errors.ts

```diff
@@ -11,32 +11,50 @@ export function throwSendEmailError(
   position: number,
   appName: string,
 ): never {
-  let stepErrorSolution = ''
   const postmanErrorData: PostmanApiErrorData = error.response.data
   const errorCode = postmanErrorData.code
   const errorMessage = postmanErrorData.message
-  // catch common errors by users and provide solution
-  if (errorCode === 'invalid_template') {
-    if (errorMessage.includes('attachments')) {
-      stepErrorSolution =
-        'Click on set up action and check that the attachment type is supported by postman. Please check the supported types at [this link](https://guide.postman.gov.sg/email-api-guide/programmatic-email-api/send-email-api/attachments#list-of-supported-attachment-file-types).'
-    } else if (errorMessage.includes('blacklisted')) {
-      stepErrorSolution =
-        'Recipient email is blacklisted. Please request for the recipient email to be whitelisted at [this link](https://guide.postman.gov.sg/api-guide/programmatic-email-api/send-email-api/recipient-blacklist).'
-    } else {
+  // catch common postman error codes and provide solution
+  switch (errorCode) {
+    case 'invalid_template':
+      if (errorMessage.includes('attachments')) {
```

Comment:
Would having a sub-error type help, instead of parsing the error message and also having to ensure that it doesn't clash with any other error messages?

Reply:
I thought that we discussed that sub-error types to just return the specific postman error e.g. `BlacklistedPostmanError` doesn't rly add value if it just returns the error message string. And can I ask how would having a sub-error type help to avoid parsing the error message here cus I'm not sure. Thanks!

Reply:
I see, I mis-interpreted the `errorMessage` variable here. My mistake, no worries!